assets/src/edit-story/components/dropzone/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { useRef } from '@wordpress/element';
+
+const DropZoneComponent = styled.div`
+	display: inherit;
+`;
+
+function DropZone( { children, onDrop } ) {
```

Comment:
I just stumbled across https://github.com/WordPress/gutenberg/pull/19514 and was wondering if we could turn this into a hook too.

Reply:
Maybe. I wonder if there could be cases where we'd benefit from the dropzone being a separate element though, e.g. if we'd want to leave space between the border of the dropzone / the actual element and add a border just to the dropzone to highlight the dropzone. Thoughts?

Reply:
> if we'd want to leave space between the border of the dropzone / the actual element and add a border just to the dropzone to highlight the dropzone

Looks like that is the case for the grid view. So I guess it makes sense the way it is now.